pkg/winutil/run_windows.go

```diff
@@ -0,0 +1,210 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Do we still need this file?

Reply:
No, I've got a change pending to remove it, just want to see how this windows e2e test run goes (in reality, its not testing anything new for this PR, I'm just curious).